helm/cluster-api-provider-aws/values.yaml

```diff
@@ -11,7 +11,7 @@ crdInstall:
   enable: true
   kubectl:
     image: "giantswarm/docker-kubectl"
-    tag: "1.23.3"
+    tag: "1.24.4"
```

Comment:
The latest v1.24.x is v1.24.10.

Reply:
```bash
➜  /Users/berkdehrioglu/gs-cluster-templates/capa docker pull giantswarm/docker-kubectl:1.24.10
Error response from daemon: manifest for giantswarm/docker-kubectl:1.24.10 not found: manifest unknown: manifest unknown
```

Reply:
Sorry, my bad, I missed that CAPA app is still using `giantswarm/docker-kubectl` and not `giantswarm/kubectl` which is more up-to-date. I had same confusion few days ago for cluster-api-app :smile: 

@fiunchinho updated cluster-api-app to use `giantswarm/kubectl` (built by regtagger, we always retag every upstream version) instead of `giantswarm/docker-kubectl` (coming from our repo, not always up-to-date) https://github.com/giantswarm/cluster-api-app/pull/82/files, I guess we can do the same here for cluster-api-provider-aws-app.

Reply:
`giantswarm/kubectl` is up-to-date

- https://hub.docker.com/layers/giantswarm/kubectl/1.24.10/images/sha256-0515444375e1ea840b6a3ea5a1197b933ba6d47326206b340e1d3f483fc331ae?context=explore
- https://quay.io/repository/giantswarm/kubectl?tab=tags